### PR TITLE
[forge] wait after logs

### PR DESCRIPTION
**Pull request by rustielin**
### Description

Some forge runs are failing because `run_forge.sh` finishes tailing the logs before kube can change the pod status from Running to Complete

wait after run_forge.sh finishes tailing the logs. sleep and `kubectl get pods` for the condition status

### Test Plan

try running the `kubectl get pods -o....` command on some running and completed pods

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2759)

### CODE REVIEW

File: testsuite/run_forge.sh [hunk 1]
```diff
@@ -242,15 +242,21 @@ elif [ "$FORGE_RUNNER_MODE" = "k8s" ]; then
     kubectl wait -n default --timeout=5m --for=condition=Ready "pod/${FORGE_POD_NAME}"
 
     # tail the logs and tee them for further parsing
+    echo "=====START FORGE LOGS====="
     kubectl logs -n default -f $FORGE_POD_NAME | tee $FORGE_OUTPUT
+    echo "=====END FORGE COMMENT====="
+
+    # wait for the pod status to change potentially
+    sleep 10
```
Comment (sitalkedia) [inline testsuite/run_forge.sh:250]: You don't need this ?

**Review by sitalkedia — COMMENTED** [review]
Do you know what changed that started causing this?

File: testsuite/run_forge.sh (hunk 1 above)
Comment (rustielin) [reply testsuite/run_forge.sh:250]: doesn't hurt to spin for another 10 seconds, but just to be safe

**Review by sitalkedia — APPROVED** [review]

**Comment by rustielin** [comment]
> Do you know what changed that started causing this?

Well it started happening after k8s 1.22 upgrade, but I doubt that changed the way `kubectl logs` interacts with the cluster.

**Pull request merged**